src/models/llama2.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from models.base import BaseChatbot, CompletionResponse
+
+class Llama2(BaseChatbot):
+
+    def __init__(
+        self,
+        prompt: str = "",
+        system_prompt: str = None,
+        api_key: str = "",
+        ) -> None:
+        self.prompt = prompt
+        self.system_prompt = system_prompt or "Answer politely"
+        self.api_key = api_key
+    
+    def get_completion(self) -> CompletionResponse:
+        API_URL = "https://api-inference.huggingface.co/models/meta-llama/Llama-2-7b-chat-hf"
+        headers = {"Authorization": f"Bearer {self.api_key}"}
+        try:
+            def query(payload):
+                 response = requests.post(API_URL, headers=headers, json=payload)
+                 return response.json()
+            
+            # Llama2 prompt template
+            prompt_template = f"<s>[INST] <<SYS>>\n{{{{ {self.system_prompt} }}}}\n<</SYS>>\n{{{{ {self.prompt} }}}}\n[/INST]"
+
+            payload = {
+                "inputs": prompt_template
+            }
+            output = query(payload)
+            
+
+        except Exception as e:
+            raise Exception("Error Occur")
+        
+        return CompletionResponse(
+            payload=output
```

Comment:
This is ok since there are default values for err and message, but we need to discuss if we should still pass in values such as empty stings